database/ddl/postgres/worker.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package postgres
+
+const (
+	// CreateWorkerTable represents a query to
+	// create the workers table for Vela.
+	CreateWorkerTable = `
+CREATE TABLE
+IF NOT EXISTS
+workers (
+	id					SERIAL PRIMARY KEY,
+	hostname			VARCHAR(250),
+	address				VARCHAR(250),
+	routes				VARCHAR(1000),
+	active     			BOOLEAN,
+	last_checked_in		INTEGER,
+	UNIQUE(hostname, address)
+);
```

Comment:
Could we fix the spacing here?

```suggestion
CREATE TABLE
IF NOT EXISTS
workers (
	id              SERIAL PRIMARY KEY,
	hostname        VARCHAR(250),
	address         VARCHAR(250),
	routes          VARCHAR(1000),
	active          BOOLEAN,
	last_checked_in INTEGER,
	UNIQUE(hostname, address)
);
```